client/app/project/project.controller.js

```diff
@@ -472,6 +427,7 @@
                 vm.resetStatusFlags();
                 vm.resetTaskData();
                 refreshProject(projectId);
+                populateMappedTaskByUserTable(projectId);
```

Comment:
Calling the reset functions happens 4x exactly the same. Would it be useful to create a function to avoid the repetition?